Kickstarter-iOS/Views/Controllers/EditorialProjectsViewController.swift

```diff
@@ -209,27 +237,21 @@ public final class EditorialProjectsViewController: UIViewController {
       self.headerTopLayoutGuide.rightAnchor.constraint(equalTo: self.headerView.rightAnchor),
       self.headerTopLayoutGuide.topAnchor.constraint(equalTo: self.headerView.topAnchor),
       headerTopLayoutGuideHeightConstraint,
-      // editorialTitleLabel
-      self.editorialTitleLabel.leftAnchor.constraint(equalTo: self.headerView.layoutMarginsGuide.leftAnchor),
-      self.editorialTitleLabel.topAnchor
-        .constraint(equalTo: self.headerTopLayoutGuide.bottomAnchor, constant: Styles.gridHalf(3)),
-      self.editorialTitleLabel.rightAnchor
-        .constraint(equalTo: self.headerView.layoutMarginsGuide.rightAnchor),
+      // editorialLabelStackView
+      self.editorialLabelStackView.trailingAnchor.constraint(equalTo: self.editorialImageView.trailingAnchor),
```

Comment:
I think we can constrain the `editorialLabelStackView` to the edges of `headerView` and then just give it top and bottom layout margins for its content.